src/Fantomas/CodePrinter.fs

```diff
@@ -481,27 +481,30 @@ and genAttribute astContext (Attribute (s, e, target)) =
     match e with
     // Special treatment for function application on attributes
     | ConstExpr (Const "()", _) ->
-        !- "[<" +> opt sepColonFixed target (!-)
+        !- "[<"
+        +> opt sepColonWithEndingSpaceFixed target (!-)
```

Comment:
I would actually go with `sepColon` in this case instead of introducing `sepColonWithEndingSpaceFixed`.

Reply:
oh sure, sorry